hooks/useKamonNFT.ts

```diff
@@ -0,0 +1,17 @@
+import { useNetwork, useSigner } from 'wagmi'
+import { ethers } from 'ethers'
+import { PodCastNFT__factory } from '@/contracts/types/factories/PodCastNFT__factory'
+import { getContractAddress } from 'utils/contractAddress'
+
+export const useKamonNFT = () => {
+  const { activeChain } = useNetwork()
+  const kamonNFT = getContractAddress({
+    name: 'kamonNFT',
+    chainId: activeChain?.id
+  })
+
+  const provider = new ethers.providers.JsonRpcProvider()
+  const { data: signer } = useSigner()
+  const contract = PodCastNFT__factory.connect(kamonNFT, signer ?? provider)
```

Comment:
use `useContract`

Reply:
I felt a little uncomfortable using useContract because the instance type is any, but it seems that if useContractWrite/Read is used, useKamonNFT does not need to be defined.

Reply:
> useContractWrtie

I saw how @geeknees was using it and it seems much better than useContract